src/wagtail_ai/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from .prompts import Prompt as PromptDataClass
+
+
+class Prompt(models.Model):
```

Comment:
**Suggestion:** Add a uuid field so the front-end can refer to a prompt without incremental primary keys that can be used for nefarious pruposes.

Reply:
I like this suggestion lots. 
Why not make the primary key an UUID?

Reply:
>  Why not make the primary key an UUID?
Django developers expect an id field to be an AutoField and you can also tweak it in your project with the `DEFAULT_AUTO_FIELD` I guess. I would not seek to complicate it unnecessarily. I'd be up for it on a one-off project, not necessarily for a library that other projects may use.

I don't think we should use auto-incremented-numerical identifiers on the front end.

Reply:
works for me

Reply:
Updated